src/go/rdctl/pkg/snapshot/manager.go

```diff
@@ -247,3 +250,13 @@ func contextIsDone(ctx context.Context) bool {
 		return false
 	}
 }
+
+// Does a utf8-aware truncation of input to maximum maxChars
+// unicode code points. Adds an ellipsis if truncation occurred.
+func truncate(input string, maxChars int) string {
```

Comment:
This one is simple, but the one in `snapshotList.go` is deceptively complex (hence the unit tests). Let's keep this one named the same, since all it is doing is truncating a string, and change the name of the other to something like `truncateForErrMsg` or something like that.

Reply:
Changed the one in `snapshotList.go` to `truncateToNewlineOrMaxRunes()`. It's long, but describes exactly what it does. Let me know if that's too goofy for you.

Reply:
I think it is fine. A bit long, but descriptive is good